### ISSUE TITLE
Shotcut runs on Mac OS X 10.9 but LSMinimumSystemVersion is 10.10

### ISSUE THREAD

**Issue by follower** [mltframework/shotcut#694]
[Shotcut (unsigned) 19.01.27 binary](https://github.com/mltframework/shotcut/releases/tag/v19.01.27) specifies that it requires Mac OS X 10.10 (a.k.a macOS) but if the `LSMinimumSystemVersion` value in the `.plist` file is changed to 10.9 it runs on Mac OS X 10.9.5[1].

Was there a particular [reason for specifying 10.10](https://github.com/mltframework/shotcut/commit/76794ea60164f15307d8489b87c78e5470af13bd) rather than 10.9?

My understanding is that [Shotcut uses Qt 5.6](https://github.com/mltframework/shotcut/blob/master/scripts/build-shotcut.sh) which officially supports [Mac OS X 10.9](https://doc.qt.io/archives/qt-5.10/supported-platforms-and-configurations.html#qt-5-6) (handled via the [`makespec`](https://doc.qt.io/qt-5/macos.html) configuration) so was there some other reason for (relatively) recently changing the minimum to 10.10?

One of the reasons why I started using Shotcut was because of its support for older OS versions and so I was disappointed when the minimum requirement went up but turned out I could've continued with it after all. :)

Even if it's not "officially" supported it'd be nice to not prevent it from running entirely.

Thanks!

[1] I didn't thoroughly test everything but I was able to import, add text & export a movie.

**Comment by ddennedy** [comment]
My oldest system is macOS 10.10, so I cannot test builds on older versions and provide support. Glad to hear that it works for you. I am in the process of upgrading Qt, and Qt 5.9 specifies 10.10 as a minimum. However, I am actually testing newer versions. I want to maintain 10.10 compatibility, and it looks like Qt 5.11.1 is the last version to support it although I might go also with 5.9.7.

**Comment by ddennedy** [comment]
I found my hackintosh HDD with OS X 10.8 on it, and changing `LSMinimumSystemVersion` does not work for me. Neither, does v19.02.20 BETA that uses Qt 5.9.7. They each crash in a different location. This will not be addressed and closing it. I was able to confirm that v19.02.20 runs on macOS 10.10, which is the minimum as stated before. Support for that version of macOS 10.10 prevents us from using the latest Qt LTS v5.12 and instead using 5.9.
